recipes/lapack95/make.inc

```diff
@@ -0,0 +1,46 @@
+#
+#  -- LAPACK95 interface driver routine (version 2.0) --
+#     UNI-C, Denmark; Univ. of Tennessee, USA; NAG Ltd., UK
+#     August 5, 2000
+#
+FC_fr	 = $(FC) -ffree-form
+FC_ff    = $(FC) -ffixed-form
+# -dcfuns  Enable recognition of non-standard double
+#          precision  complex intrinsic functions
+# -dusty   Allows the compilation and execution of "legacy"
+#          software by downgrading the category  of  common
+#          errors  found  in  such software from "Error" to
+# -ieee=full enables all IEEE arithmetic facilities
+#          including non-stop arithmetic.
+
+OPTS0    = -pipe
+MODLIB   = -I$(PREFIX)/include
+OPTS1    = -c -O3 $(OPTS0)
+OPTS3    = $(OPTS1) $(MODLIB)
+OPTL     = -o 
+OPTLIB   =
+
+LAPACK_PATH = $(PREFIX)/lib
+
+LAPACK95 = ../lapack95.a
```

Comment:
```suggestion
LAPACK95 = ../lapack95$SHLIB_EXT
```

Reply:
`SHLIB_EXT` will make it be `.so`, right? If yes, I can't do it as the makefile has the hardcoded name to generate the file as `lapack95.a`.
Here:
![image](https://user-images.githubusercontent.com/8185425/110844869-cba42100-825e-11eb-9cac-f14ca6c13cf3.png)

Reply:
We can add and apply a patch to fix that

https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#patches

Reply:
I am familiar with the possibility of applying patches to packages I don't think it is right to name a static library as `x.so`.
Is it the case that unless I rewrite their makefile or patch it to produce a shared library this recipe won't be merged?

If that is the only solution that's fine but I would like to avoid going this route if possible. Please don't get me wrong it is just that this is a library that no one touched since November 2000 and I see little to no benefit on the change other than the reasoning from CFEP-18.

Let me know how you would like me to proceed so this can be merged.

Reply:
@hhslepicka, you can create a shared library from static library without having to rewrite the makefile.
One problem with static builds would be that if you use this package in a downstream package, then you'll have to package the license of this package in the downstream package which makes it messy.

Reply:
@isuruf could you share how to do this (create the shared from the static) without rewriting the makefile?

Reply:
after building the static library, do the following,
```bash
if [[ "$target_platform" == linux-* ]]; then
  $FC  -shared -o -Wl,--whole-archive  liblapack95.so  -Wl,--no-whole-archive $PREFIX/lib/liblapack.so $PREFIX/lib/libblas.so 
elif [[ "$target_platform" == osx-* ]]; then
  $FC -shared -o  -Wl,-all_load  liblapack95.dylib -Wl,-noall_load $PREFIX/lib/liblapack.dylib $PREFIX/lib/libblas.dylib 
fi
```
It's untested, so there might be typos above

Reply:
I will give it a try here and get back to you on that...

Reply:
@isuruf I am not getting where the static lib goes into play on the generation of the shared one there. It does not use `liblapack95.a` anywhere.